### PR TITLE
launchpad msr

### PR DESCRIPTION
**Pull request by james-nesbitt**
- launchpad now installs msr
	- ingress for msr
	- firewall for msr (permissive)
- clean up mke/msr flags across provision and yaml output

ALSO

- better comments in the tfvars tpl
- more default tags for user/expiry

TODO
- improve the MSR firewall
- fix the MSR lb ingress

### CODE REVIEW

File: launchpad-stack-aws3/infra/launchpad.tf [hunk 1]
```diff
@@ -126,11 +126,31 @@ resource "launchpad_config" "cluster" {
...
+      install_flags = [
+        "--san=${local.MKE_URL}",
+      ]
+    }
+
+    dynamic "msr" {
+      for_each = local.has_msr ? [1] : []
+
+      content {
+        version    = var.launchpad.mke_version
```
Comment (pgedara) [inline launchpad-stack-aws3/infra/launchpad.tf:147]: I think this should be msr_version?

Comment (james-nesbitt) [reply launchpad-stack-aws3/infra/launchpad.tf:147]: yes

File: launchpad-stack-aws3/infra/launchpad.tf [hunk 2]
```diff
@@ -126,11 +126,31 @@ resource "launchpad_config" "cluster" {
...
+        "--san=${local.MKE_URL}",
+      ]
+    }
+
+    dynamic "msr" {
+      for_each = local.has_msr ? [1] : []
+
+      content {
+        version    = var.launchpad.msr_version
+        image_repo = "docker.io/Mirantis"
```
Comment (pgedara) [inline launchpad-stack-aws3/infra/launchpad.tf:148]: Repo name should be in lowercase (mirantis, instead of Mirantis); If not, launchpad throws an error.

Comment (james-nesbitt) [reply launchpad-stack-aws3/infra/launchpad.tf]: fixed

**Comment by james-nesbitt** [comment]
I added an LB and an SG for MSR to see if that fixes things.

**Pull request merged**